faq.md

```diff
@@ -0,0 +1,36 @@
+# FAQ
+
```

Comment:
Like I wrote above in the issue description, I also would like to add a table of content. I haven't got time yet to look how to do that. I thought it would be good to show you the FAQ content asap and add the table stuff in the meantime while we are discussing the content. 
But thanks so far :), I take a deeper look into the posibilities of jekyll.